### PR TITLE
Placeholder Fix for Input Date and Input Time

### PR DESCRIPTION
**Pull request by dgharde**
## Related Issue
Please use one of the well-known [github fixes keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to reference
the issues fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one if reasonable to aid 
in issue tracking.

**NOTE**: For multiple issues resolved by this PR use the corresponding keywords **every time** in a comma-delimited list per the reference
page above.

## Description
- Changed the separator between day, month an year in InputDate to "/" from "-"
- Fix for the placeholder bug where placeholder does not reappear once the Input Date or Input Time gains focus 

## Sample Card
Input Date 
image

https://user-images.githubusercontent.com/78541663/112466227-a0a3ec00-8d8b-11eb-8d46-3801900a4d42.mp4

## How Verified
How you verified the fix, including one or all of the following: 
1. New unit tests that were added if any. If none were added please add a quick line explaining why not.
2. Existing relevant unit/regression tests that you ran
3. Manual scenario verification if any; ***Do include .gif's or screenshots of the testing you performed here if you think that it 
will aid in code reviews or corresponding fixes on other platforms for eg.***

### CODE REVIEW

File: source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp [hunk 1]
```diff
@@ -1238,7 +1238,7 @@ namespace RendererQml
 		auto month_Text = "getText(0,3)";
 		auto day_Text = "getText(4,6)";
 		auto year_Text = "getText(7,11)";
-		std::string DateRegex = Formatter() << "/^" << month_Regex << dateSeparator << day_Regex << dateSeparator << year_Regex << "$/";
+		std::string DateRegex = Formatter() << "/^" << month_Regex << "\\" << dateSeparator << day_Regex << "\\" << dateSeparator << year_Regex << "$/";
```
Comment (rj100189) [inline source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp:1241]: can we not do `const auto dateSeparator = "\\/";` instead of adding everywhere?

**Review by rj100189 — CHANGES_REQUESTED** [review]

File: source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp (hunk 1 above)
Comment (dgharde) [reply source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp]: Yeah this works. Thanks for the suggestion.
I have added this change in the latest commit

File: source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp [hunk 2]
```diff
@@ -1238,7 +1238,7 @@ namespace RendererQml
 		auto month_Text = "getText(0,3)";
 		auto day_Text = "getText(4,6)";
 		auto year_Text = "getText(7,11)";
-		std::string DateRegex = Formatter() << "/^" << month_Regex << dateSeparator << day_Regex << dateSeparator << year_Regex << "$/";
+		std::string DateRegex = Formatter() << "/^" << month_Regex << dateSeparator << day_Regex  << dateSeparator << year_Regex << "$/";
```
Comment (rj100189) [inline source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp:1241]:
```suggestion
		std::string DateRegex = Formatter() << "/^" << month_Regex << dateSeparator << day_Regex << dateSeparator << year_Regex << "$/";
```

File: source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp [hunk 3]
```diff
@@ -1248,7 +1248,7 @@ namespace RendererQml
 			{
 				StringDateFormat = Formatter() << "dd" << dateSeparator << "MMM" << dateSeparator << "yyyy";
 				inputMask = Formatter() << "xx" << dateSeparator << ">x<xx" << dateSeparator << "xxxx;-";
-				DateRegex = Formatter() << "/^" << day_Regex << dateSeparator << month_Regex << dateSeparator << year_Regex << "$/";
+				DateRegex = Formatter() << "/^" << day_Regex  << dateSeparator << month_Regex  << dateSeparator << year_Regex << "$/";
```
Comment (rj100189) [inline source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp:1251]:
```suggestion
				DateRegex = Formatter() << "/^" << day_Regex << dateSeparator << month_Regex << dateSeparator << year_Regex << "$/";
```

File: source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp [hunk 4]
```diff
@@ -1259,7 +1259,7 @@ namespace RendererQml
 			{
 				StringDateFormat = Formatter() << "yyyy" << dateSeparator << "MMM" << dateSeparator << "dd";
 				inputMask = Formatter() << "xxxx" << dateSeparator << ">x<xx" << dateSeparator << "xx;-";
-				DateRegex = Formatter() << "/^" << year_Regex << dateSeparator << month_Regex << dateSeparator << day_Regex << "$/";
+				DateRegex = Formatter() << "/^" << year_Regex  << dateSeparator << month_Regex  << dateSeparator << day_Regex << "$/";
```
Comment (rj100189) [inline source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp:1262]:
```suggestion
				DateRegex = Formatter() << "/^" << year_Regex << dateSeparator << month_Regex << dateSeparator << day_Regex << "$/";
```

File: source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp [hunk 5]
```diff
@@ -1270,7 +1270,7 @@ namespace RendererQml
 			{
 				StringDateFormat = Formatter() << "yyyy" << dateSeparator << "dd" << dateSeparator << "MMM";
 				inputMask = Formatter() << "xxxx" << dateSeparator << "xx" << dateSeparator << ">x<xx;-";
-				DateRegex = Formatter() << "/^" << year_Regex << dateSeparator << day_Regex << dateSeparator << month_Regex << "$/";
+				DateRegex = Formatter() << "/^" << year_Regex  << dateSeparator << day_Regex  << dateSeparator << month_Regex << "$/";
```
Comment (rj100189) [inline source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp:1273]:
```suggestion
				DateRegex = Formatter() << "/^" << year_Regex << dateSeparator << day_Regex << dateSeparator << month_Regex << "$/";
```

**Review by rj100189 — CHANGES_REQUESTED** [review]

**Review by rj100189 — APPROVED** [review]

**Pull request merged**